src/ga4gh/core/_internal/enderef.py

```diff
@@ -73,15 +82,18 @@ def ga4gh_deref(o, cra_map, object_store):
     def _deref(o):
         """depth-first recursive, in-place deref of object; returns id of object"""
         if o.type not in cra_map:
+            _logger.warn(f"{o.type} not in cra_map {cra_map}")
```

Comment:
warn is deprecated, see [here](https://docs.python.org/3/library/logging.html#logging.Logger.warning)
```suggestion
            _logger.warning(f"{o.type} not in cra_map {cra_map}")
```